rosa/cli.py

```diff
@@ -131,8 +131,9 @@ def execute(command, allowed_commands=None):
                     }}
 
     Returns:
-        list or json: json if json.loads(res.stdout) not fail,
-        else list of output line by line.
+        json or dict: json if json.loads(res.stdout) not fail,
```

Comment:
pleas rephrased, you always return dict
I think we always need to return the same structure in the dict
{"stdout": ......, "stderr": .....}

Reply:
{"out": ......, "err": .....}